File: Coursework/Level 4/Refactoring Task/NHIF.java
public class NHIFSystem {

    // Global-style variables (Data)
    static String patientName = "Jaymin";
    static String policyNumber = "NHIF-B123";

    // Standalone logic (Function)
    public static double calculateClaim(double amount) {
        double coPayment = amount * 0.10;
        return amount - coPayment;
    }

    public static void main(String[] args) {
        double bill = 12000.0;
        
        // We call the function directly without 'new Patient()'
        double payout = calculateClaim(bill);
        
        System.out.println("Patient: " + patientName);
        System.out.println("NHIF Payout: " + payout + " KES");
    }
}

File: Coursework/Level 4/Refactoring Task/NHIF.py
def calculate_claim(amount):
    """
    Deducts 10% co-payment from the total bill.
    """
    co_payment = amount * 0.10
    return amount - co_payment

patient_data = {
    "name": "Jaymin",
    "policy_number": "NHIF-778899"
}

bill = 5000
payout = calculate_claim(bill)

print(f"Processing for: {patient_data['name']}")
print(f"NHIF Payout: {payout} KES")